docs/run.md

```diff
@@ -144,19 +144,15 @@ functionality.
 ### List Network Changes
```

Comment:
maybe worth putting in a comment saying that if you use defaults certs these path are ok, otherwise please use --certPath and --keyPath flags.